### PR TITLE
Fix env load problems in scripts.

### PR DESCRIPTION
**Pull request by mccalluc**
@afred: review.

### CODE REVIEW

**Comment by mccalluc** [comment]
cancel: This should go into 1.x

File: script/import_metadata [hunk 1]
```diff
@@ -16,7 +16,7 @@ end
 
 
 puts "Loading environment..."
-require File.join(File.dirname(__FILE__), '../config/environment.rb')
+require_relative('../config/environment')
```
Comment (afred) [inline script/import_metadata]: This will only work if you're in some directory that is one deeper than `Rails.root`. Is that what you're going for?

Comment (mccalluc) [inline script/import_metadata]: The old one wasn't working; the new one works as long the script files remain in the same relative position. If they need to move, the path can be tweaked.

These scripts don't have a notion of rails root, so there is nothing to start a path from except their own location.

Comment (afred) [inline script/import_metadata]: I think the most flexible is something like this:
```ruby
File.join(File.expand_path(File.dirname(__FILE__)), '..', 'config', 'environment')
```
`File.join` will ensure you're using the correct directory separator, e.g. `/`.
`File.expand_path` will give you an absolute path.
`File.dirname(__FILE__)` will give you the directory name of the _current file_, so you'll get the right dir no matter where you are calling the script from.

It's stupidly verbose, each method does something different.

**Pull request closed without merging**